apps/backoffice/src/types/selfcare/AckPayloadRequest.ts

```diff
@@ -0,0 +1,21 @@
+/**
```

Comment:
Are all those generated modules required and used in the project?

Reply:
No, they aren't. The only way to avoid generating unused types is to manually modify selfcare OpenAPI, that might be error pruned and also, in some cases, hard to find all model and sub model related to the "target" endpoint we want to punctually integrate.